docs/src/howto/storage.md

```diff
@@ -0,0 +1,36 @@
+# Using Default Storage in Canonical Kubernetes
+
+Canonical Kubernetes offers a local storage option to quickly set up and run a cluster, especially for single-node support. This guide walks you through enabling and configuring this feature.
+
+Before you start, ensure you have a bootstrapped Canonical Kubernetes cluster ready.
+Check out the [Getting Started](https://github.com/canonical/k8s-snap/blob/main/docs/src/tutorial/getting-started.md) guide to learn how to do this.
+
+### 1. Enable Storage
+When bootstrapping the snap, the storage functionality is not enabled by default. To enable it, execute the following command:
+
+```sh
+sudo k8s enable storage
+```
+
+### 2. Configure Storage
+While the storage option comes with sensible defaults, you can customize it to meet your requirements. Obtain the current configuration by running:
+
+```sh
+sudo k8s get storage
+```
+
+> **Note**: For an explanation of each configuration option, refer to the [reference](#TODO) section. 
```

Comment:
N.B. We can use sphinx-style admonitions now 
```
``` {note} xxxx xxxxxx ```
```

In this case though I think we should discuss the options here. There is no wider reference doc to refer to